ibm/provider/provider.go

```diff
@@ -1058,6 +1058,7 @@ func Provider() *schema.Provider {
 			// //Added for Transit Gateway
 			"ibm_tg_gateway":                  transitgateway.ResourceIBMTransitGateway(),
 			"ibm_tg_connection":               transitgateway.ResourceIBMTransitGatewayConnection(),
+			"ibm_tg_connection_actions":       transitgateway.ResourceIBMTransitGatewayConnectionActions(),
```

Comment:
Will this resource create multiple actions?

If not, can we name it to `ibm_tg_connection_action`?
and also change the name accordingly in all the references, filenames and in docs.

Reply:
I renamed it to 'ibm_tg_connection_action' and updated all references and docs.